en/theme/material/assets/css/theme.css

```
@@ -0,0 +1,942 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+ 
+@import url('partials/_typography.css');
```

Stylelint (17.14.0)

[error] Expected "url('partials/_typography.css')" to be "'partials/_typography.css'" (import-notation)

(import-notation)